test/core/interface.spec.js

```diff
@@ -16,7 +16,14 @@ describe('interface-ipfs-core tests', () => {
 
   tests.config(defaultCommonFactory)
 
-  tests.dag(defaultCommonFactory)
+  tests.dag(defaultCommonFactory, {
+    skip: [
+      {
+        name: 'should override hash algoritm default and resolve with it',
+        reason: 'ipld/js-ipld/pull/133 needs to land first'
```

Comment:
Could you please add the full URL here? CI actually turns these into clickable links e.g.

<img width="1110" alt="screen shot 2018-07-05 at 08 58 19" src="https://user-images.githubusercontent.com/152863/42309788-89b62c48-8031-11e8-845b-b2ca9fe7cf76.png">

Reply:
Done! :)